.github/workflows/deploy.yml

```
@@ -53,8 +53,8 @@ jobs:
       run: |
         git config user.name "pytest bot"
         git config user.email "pytestbot@gmail.com"
-        git tag --annotate --message=v${{ github.event.inputs.version }} v${{ github.event.inputs.version }} ${{ github.sha }}
-        git push origin v${{ github.event.inputs.version }}
+        git tag --annotate --message=v${{ github.event.inputs.version }} ${{ github.event.inputs.version }} ${{ github.sha }}
+        git push origin ${{ github.event.inputs.version }}
```

actually the v prefix should stay - its intentional to ensure the tags dont start with numbers
```suggestion
        TAG_VERSION="v${{ github.event.inputs.version }}"
        git tag --annotate \
          --message="Release ${TAG_VERSION}" \
          "${TAG_VERSION}" ${{ github.sha }}
        git push origin $TAG_VERSION
```

---

Are you sure? This is different from all other tags we have in the repository:

```
λ git tag
...
1.0.1
1.0.2
1.1.0
1.1.1
1.2.0
1.2.1
1.3.0
...
7.4.0
7.4.0.dev0
7.4.1
7.4.2
7.4.3
8.0.0.dev0
v7.4.3
```

Do we want to change that?

---

I'm a bit torn between leaving things as is and using the new style in future

---

I'm with @bluetech.